### PR TITLE
Cu 86b14rync setup pages UI first

### PR DESCRIPTION
**Pull request by Victorportus**
- Welcome page UI
- Phone setup UI

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b14rync Setup Pages UI](https://app.clickup.com/t/86b14rync)

**Review by whamester — APPROVED** [review]

**Comment by whamester** [comment]
rename files:

phone-training.tsx ->  phone-mode-confirmation.tsx
phone1.tsx -> phone-training.tsx

**Review by ReinhardtBotha — APPROVED** [review]
Looks good Victor!

File: src/app/setup/enable-notifications.tsx [hunk 1]
```diff
@@ -1,48 +0,0 @@
-import Main from "@src/components/layout/Main";
-import Button from "@src/components/ui/Button";
```
Comment (whamester) [inline src/app/setup/enable-notifications.tsx:2]: we still need this screen:
image

File: src/styles/globalStyles.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const globalStyles = {
       50: "#FFF8E6",
       100: "#FFF1CC",
       200: "#FEE39A",
-      300: "#FDD462",
+      300: "#FFDF88",
```
Comment (whamester) [inline src/styles/globalStyles.ts:12]: thank you for this fix

**Review by zeckkimon — APPROVED** [review]

File: src/app/setup/enable-notifications.tsx (hunk 1 above)
Comment (Victorportus) [reply src/app/setup/enable-notifications.tsx:2]: I will add it back on the next PR
TKS

**Pull request merged**